Lib/test/test_copyreg.py

```diff
@@ -32,16 +35,15 @@ class WithInherited(WithSingleString):
 class CopyRegTestCase(unittest.TestCase):
 
     def test_class(self):
-        self.assertRaises(TypeError, copyreg.pickle,
-                          C, None, None)
+        copyreg.pickle(C, pickle_C)
```

Comment:
This change looks like a mistake.  The next two are not clear without knowing more about copyreg.pickle.

Reply:
This test was initially intended  for testing TypeError when register pickling for classic classes. But it was broken in Python 3: a TypeError was raised not because C was a classic class (there are no classic classes in Python 3), but because the second argument is not callable. This case is covered by the following test.

This test could be removed, but I decided to repurpose it, because there were no positive tests for `copyreg.pickle()`.

Reply:
This change is unrelated to the other changes in this PR (the change itself looks sane)

Reply:
In any case, this change is part of the merged patch, which is okay with me given Serhiy's response.